src/components/ProjectGroup/ProjectGroup.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './ProjectGroup.css';
+import ProjectCard from '../ProjectCard/ProjectCard'
+
+function ProjectGroup({name, projects}) {
+  return (
+    <div className="ProjectGroup">
+      <h2 className="ProjectGroup-Name">{name}</h2>
+      <div className="ProjectCards">
```

Comment:
__*Nice to have*__
Another small accessibility thing this is a list so it should probably be a `ul` and each project card wrapped in an `li` (that is required by html spec)